mkosi/__init__.py

```diff
@@ -4054,12 +4054,17 @@ def install_unified_kernel(
                 image_id = f"mkosi-{args.distribution}"
                 partlabel = None
 
+            # See https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/
```

Comment:
https://systemd.io/BOOT_LOADER_SPECIFICATION/#boot-counting is a better link.

Reply:
Done